WORKSPACE.yaml

```diff
@@ -7,7 +7,7 @@ defaultArgs:
   jbMarketplacePublishTrigger: "false"
   publishToJBMarketplace: true
   localAppVersion: unknown
-  codeCommit: 971d2f15e8df4000a19e45387f48100add465ce7
+  codeCommit: 1474ed291d29192401d8e4eebcd10b78da7b7ed4
```

Comment:
Could you replace the commit hash to code base that aligns to https://github.com/gitpod-io/gitpod/pull/11361#discussion_r926410104 so that we can test it in preview env?

Reply:
Should be good now :)